dashboard/lib/policies/child_account.rb

```diff
@@ -89,6 +100,9 @@ def self.lockout_date(user)
   end
 
   private_class_method def self.state_policy(user)
+    # If the country_code is not set, then us_state value was inherited
+    # from the teacher and we don't trust it.
+    return unless user.country_code
```

Comment:
What's the rationale for putting this here instead of in `self.user_predates_policy?` It seems like if we contain all these changes to a single function, it's easier to remove/disable them later.

Reply:
I put it here because this is the only place that `user.us_state` is read in this file. And there is already logic in other parts of the code to handle the case where state_policy is `nil`.

Reply:
Makes sense - I'm good with it staying here as long as we have a plan to remove it once we hit phase 2